rust-toolchain

```diff
@@ -1 +1 @@
-1.56.0
+1.57.0
```

Comment:
required by some dependency ; check the previous commit test failure if you need more info